### PR TITLE
Add a feature getting user's email on KAKAO provider

### PR DESCRIPTION
**Pull request by JooeunAhn**
Today, KAKAO API start to provide the user's email information.
So, I added a feature that gets the email from kakao api

### CODE REVIEW

File: allauth/socialaccount/providers/kakao/provider.py [hunk 1]
```diff
@@ -23,5 +24,24 @@ class KakaoProvider(OAuth2Provider):
...
+        if data.get("kaccount_email_verified"):
+            email = data.get("kaccount_email")
+
+        return dict(email=email)
+
+    def extract_email_addresses(self, data):
+        ret = []
+        email = data.get("kaccount_email")
+        verified = data.get("kaccount_email_verified")
+        if verified:
```
Comment (pennersr) [inline allauth/socialaccount/providers/kakao/provider.py]: Can you change this so that the email address is also added when verified is not true?

**Comment by JooeunAhn** [comment]
@pennersr Of course. Does this change you want?

**Comment by JooeunAhn** [comment]
@pennersr Of course!  Is this the change you want?

**Comment by pennersr** [comment]
Thanks

**Pull request merged**